src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.number = getRandomNumber();
+        ball.color = String.valueOf(new ColorSupplier().getRandomColor());
+        return ball;
+    }
+
+    public int getRandomNumber() {
+        Random number = new Random();
+        int value = number.nextInt(100);
+        return value;
+    }

```

Comment:
I think this method should be private as we use it only in this class. And lets make Random as class level variable. Also lets move number 100 to constant https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names